Source/Scene/PostProcessStageLibrary.js

```diff
@@ -0,0 +1,663 @@
+                },
+                set : function(value) {
+                    blur.uniforms.stepSize = value;
+                }
+            },
+            ambientOcclusionOnly : {
+                get : function() {
+                    return ambientOcclusionModulate.uniforms.ambientOcclusionOnly;
+                },
+                set : function(value) {
+                    ambientOcclusionModulate.uniforms.ambientOcclusionOnly = value;
+                }
+            }
+        });
+
+        return new PostProcessStageComposite({
+            name : 'czm_ambient_occlusion',
+            stages : [generateAndBlur, ambientOcclusionModulate],
+            inputPreviousStageTexture : false,
+            uniforms : uniforms
+        });
+    };
+
+    var fxaaFS =
+        '#define FXAA_QUALITY_PRESET 39 \n' +
+        FXAA3_11 + '\n' +
+        FXAA;
+
+    /**
+     * Creates a post-process stage that applies Fast Approximate Anti-aliasing (FXAA) to the input texture.
+     * @return {PostProcessStage} A post-process stage that applies Fast Approximate Anti-aliasing to the input texture.
+     *
+     * @private
+     */
+    PostProcessStageLibrary.createFXAAStage = function() {
+        return new PostProcessStage({
+            name : 'czm_FXAA',
+            fragmentShader : fxaaFS,
+            sampleMode : PostProcessStageSampleMode.LINEAR
+        });
+    };
+
+    /**
+     * Creates a post-process stage that renders the input texture with black and white gradations.
+     * <p>
+     * This stage has one uniform value, <code>gradations</code>, which scales the luminance of each pixel.
+     * </p>
+     * @return {PostProcessStage} A post-process stage that renders the input texture with black and white gradations.
+     */
+    PostProcessStageLibrary.createBlackAndWhiteStage = function() {
+        return new PostProcessStage({
+            name : 'czm_black_and_white',
+            fragmentShader : BlackAndWhite,
+            uniforms : {
+                gradations : 5.0
+            }
+        });
+    };
+
+    /**
+     * Creates a post-process stage that saturates the input texture.
+     * <p>
+     * This stage has one uniform value, <code>brightness</code>, which scales the saturation of each pixel.
+     * </p>
+     * @return {PostProcessStage} A post-process stage that saturates the input texture.
+     */
+    PostProcessStageLibrary.createBrightnessStage = function() {
+        return new PostProcessStage({
+            name : 'czm_brightness',
+            fragmentShader : Brightness,
+            uniforms : {
+                brightness : 0.5
+            }
+        });
+    };
+
+    /**
+     * Creates a post-process stage that transforms the input texture to a pixelated eight-vit style image.
+     * @return {PostProcessStage} A post-process stage that transforms the input texture to a pixelated eight-vit style image.
+     */
+    PostProcessStageLibrary.createEightBitStage = function() {
```

Comment:
I like this one but, in practice, should we keep it in core Cesium?

Or a few years from now, will it end up in something forgotten like the https://github.com/AnalyticalGraphicsInc/cesium-materials-pack

Could be better of as a Sandcastle example of creating a custom post-processing stage.